docs/troubleshooting.md

```diff
@@ -56,25 +59,23 @@ completed the compilation successfully before loading a view.
 * If you get this error while trying to use Elm, try rebuilding Elm. You can do
   so with a postinstall hook in your `package.json`:
 
-```
+```json
 "scripts": {
   "postinstall": "npm rebuild elm"
 }
 ```
 
-
 ## webpack or webpack-dev-server not found
 
-* This could happen if  `webpacker:install` step is skipped. Please run `bundle exec rails webpacker:install` to fix the issue.
+* This could happen if `webpacker:install` step is skipped. Please run `bundle exec rails webpacker:install` to fix the issue.
 
 * If you encounter the above error on heroku after upgrading from Rails 4.x to 5.1.x, then the problem might be related to missing `yarn` binstub. Please run following commands to update/add binstubs:
 
 ```bash
-bundle config --delete bin
-./bin/rails app:update:bin # or rails app:update:bin
+$ bundle config --delete bin
+$ ./bin/rails app:update:bin # or rails app:update:bin
```

Comment:
There is a slight downside to including `$` as a prompt in bash examples. It is more difficult to copy/paste the example. Not a big deal, just something I've noticed recently.

Reply:
I agree with @jaredbeck. What's the upside of the `$`? Many people use fancy prompts without the `$`.

Reply:
I find it easier to see the beginning of the command. I can remove it if needed

Reply:
> I can remove it if needed

It's a lot easier to copy/paste without it, thanks :)